cast/java/src/testFixtures/java/com/ibm/wala/cast/java/test/IRTests.java

```diff
@@ -333,7 +335,7 @@ public void check(CallGraph cg) {
               }
             }
 
-            assertFalse(false, "cannot find " + at);
+            Assertions.fail("cannot find " + at);
```

Comment:
Just `fail(...)` due to static import?

Reply:
Good catch. I'll change that in the next update to this PR. I looked for other instances of `Assertions.fail` but found none.